src/Plutarch/Orphans.hs

```diff
@@ -103,6 +108,22 @@ instance PTryFrom PData (PAsData PScriptHash) where
 
         pure (punsafeCoerce opq, pcon $ PScriptHash unwrapped)
 
+----------------------------------------
+-- Instances for Ratios
+
+instance ToData (Ratio Integer) where
+    toBuiltinData rat =
+        BuiltinData $
+            List
+                [ I $ numerator rat
+                , I $ denominator rat
+                ]
+
+instance FromData (Ratio Integer) where
+    fromBuiltinData (BuiltinData (List [I num, I denom])) =
+        pure $ num % if num == 0 then 1 else denom
```

Comment:
Why is the `if` necessary here?

Reply:
[See here](https://liqwidlabs.slack.com/archives/C01R16YV5EV/p1663779182508699?thread_ts=1663778686.919799&cid=C01R16YV5EV).

Short version: this may or may not still be necessary. We should check, and add to the documentation with the results.